ocean_lib/ocean/test/test_ocean_assets.py

```diff
@@ -594,3 +594,37 @@ def test_compressed_and_encrypted_asset(
     file2_dict = {"type": "url", "url": "https://url.com/file2.csv", "method": "GET"}
     assert file1_dict in asset_urls
     assert file2_dict in asset_urls
+
+
+def test_asset_creation_errors(publisher_ocean_instance, publisher_wallet, config):
+    erc721_token, erc20_token = deploy_erc721_erc20(
+        publisher_ocean_instance.web3, config, publisher_wallet, publisher_wallet
+    )
+
+    web3 = publisher_ocean_instance.web3
+    data_provider = DataServiceProvider
+    _, metadata, encrypted_files = create_basics(config, web3, data_provider)
+
+    # used a random address from Etherscan
+    some_random_address = "0xB3b8239719403E38de3bdF19B9AC147B48c72BF2"
```

Comment:
If you want the contract to definitely never be found, it might be better to just use the zero address